### PR TITLE
Simplified Chinese Translation 

### PR DESCRIPTION
**Pull request by dingyifei**
#130 
https://en.wikipedia.org/wiki/Template:ISO_639_name

Maybe it should be renamed to zh-s.ini or zhs.ini?

### CODE REVIEW

**Comment by Aericio** [comment]
Yeah, rename them to zh-s and zh-t please. Thanks.

**Comment by dingyifei** [comment]
I renamed zho.ini and chi.ini to zht.ini and zhs.ini

**Comment by dingyifei** [comment]
I see #138 had changed zho.ini, should I revert my 7db2ece commit?

**Comment by Aericio** [comment]
No, since you're first we'll accept yours and reject his.

**Comment by inxomnyaa** [comment]
I guess it doesn't matter that much - it is just a guideline, but the filename is never shown anywhere else than github, neither exposed in the plugin - what matters is the language.name string :P 
The iso639-2 format was just so people can easier recognize the files they have to edit, since we had duplicated languages last year with different file names

File: resources/lang/zht.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ; Updated time : 26th 07 2017
-language.name = "Chinese"
+language.name = "Traditional Chinese"
```
Comment (inxomnyaa) [inline resources/lang/zht.ini]: Good point to change it 👍

**Comment by dingyifei** [comment]
I revert the cht.ini and chs.ini back to zho.ini and chi.ini based on user4086's comment on that StackExchange question

File: resources/lang/zho.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ; Updated time : 26th 07 2017
-language.name = "Traditional Chinese"
```
Comment (inxomnyaa) [inline resources/lang/zho.ini]: wasn't this changed to remove confusion?

Comment (dingyifei) [reply resources/lang/zho.ini]: I revert the change, then changed zho.ini to "Traditional Chinese".... right now the "zho.ini" should be Traditional Chinese and the "chi.ini" is "Simplified Chinese". I just realized I did the opposite of what the post said...

**Comment by dingyifei** [comment]
I switch the zho.ini and chi.ini, so now Simplified Chinese = "zho.ini" and Traditional Chinese = "chi.ini" as the post said.

>  I'll bet that the designer takes "zho" to mean simplified Chinese and "chi" to mean traditional Chinese. Reason: "zhong" is a mainland-style "Pinyin" transcription, and in regions where traditional Chinese is used (such as Taiwan and Hong Kong), "Pinyin" is not the default phonetic system. – user4086

**Comment by inxomnyaa** [comment]
Ok cool 👍 😊

**Pull request merged**